RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -43,6 +43,8 @@ def TICL_iterations_withReco(process):
   )
 
   process.FilteredLayerClusters = filteredLayerClustersProducer.clone(
+      clusterFilter = "ClusterFilterByAlgoAndSize",
```

Comment:
this is already a part of filteredLayerClustersProducer PSet. Is this redundant change really necessary here?